app/views/cloud_subnet/edit.html.haml

```diff
@@ -0,0 +1,68 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "cloudSubnetFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Edit Subnet')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet Name')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "name",
+                            'ng-model'     => "cloudSubnetModel.name",
+                            'ng-maxlength' => 128,
+                            :miqrequired   => true,
+                            :checkchange   => true}
+    .form-group
+      %label.col-md-2.control-label
+        = _('CIDR')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "cidr",
+                            'ng-model'     => "cloudSubnetModel.cidr",
+                            'ng-maxlength' => 20,
+                            :miqrequired   => false,
+                            :checkchange   => true}
+    .form-group
+      %label.col-md-2.control-label
+        = _('Gateway')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "ip_version",
+                            'ng-model'     => "cloudSubnetModel.gateway",
+                            'ng-maxlength' => 20,
+                            :miqrequired   => false,
+                            :checkchange   => true}
+    .form-group
+      %label.col-md-2.control-label
+        = _('IP Version')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "ip_version",
+                            'ng-model'     => "cloudSubnetModel.ip_version",
+                            'ng-maxlength' => 2,
+                            :miqrequired   => true,
+                            :checkchange   => true}
+
+
+  %table{:width => '100%'}
```

Comment:
Same here, you may want to use `= render :partial => "layouts/angular/x_edit_buttons_angular"` here instead of the buttons. (As they are now, they all work even when disabled ;))